util/clientmetric/clientmetric.go

```diff
@@ -223,6 +224,33 @@ func NewGaugeFunc(name string, f func() int64) *Metric {
 	return m
 }
 
+type AggregateCounter struct {
+	mu       sync.RWMutex
+	counters []*expvar.Int
+}
+
+func (c *AggregateCounter) Value() int64 {
```

Comment:
Probably worth adding at least a simple test confirming that it works correctly with 0, 1 and 2 registered vars.